src/navigation/MainNavigator.tsx

```diff
@@ -23,6 +27,11 @@ const MainNavigator: FunctionComponent = () => {
         ) : (
           <Stack.Screen name={Stacks.Onboarding} component={OnboardingStack} />
         )}
+        <Stack.Screen
```

Comment:
@johnschoeman I hate that we have to do this for any "modal" screens, but it's the only way to get it to work properly. Is there a better way to organize this? I don't like how this is going to scale.

Reply:
Yup I think we'll want a `Modal` stack of some sort: https://reactnavigation.org/docs/modal/

which not 100% sure, but likely wants to live in the 'MainNavigator' bit. so something like:

```
<NavigationContainer>
{ hasCompletedOnboarding ?
  <Navigator.Stack mode={Modal}>
      <MainTabNavigator />
      <Modal 1 />
      <Modal 2/>
  </Navigator.Stack>
  : <OnboardingScreen>
}
</NavigationContainer>
```